Source/Core/Core/BootManager.cpp

```diff
@@ -333,6 +333,8 @@ bool BootCore(const std::string& _rFilename)
                          StringFromFormat("Movie%s.raw", (i == 0) ? "A" : "B")))
           File::Delete(File::GetUserPath(D_GCUSER_IDX) +
                        StringFromFormat("Movie%s.raw", (i == 0) ? "A" : "B"));
+        if (File::Exists(File::GetUserPath(D_GCUSER_IDX) + "Movie"))
```

Comment:
Would it make sense to store the directory path instead of duplicating it, i.e.:

    const std::string movie_directory = File::GetUserPath(D_GCUSER_IDX) + "Movie");
    if (File::Exists(movie_directory))
      File::DeleteDirRecursively(movie_directory);

This is less error prone, imo.